src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -147,12 +155,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 if ($this->defaultTransPath) {
                     $transPaths[] = $this->defaultTransPath.'/'.$bundle->getName();
                 }
-                $transPaths[] = sprintf('%s/Resources/%s/translations', $kernel->getRootDir(), $bundle->getName());
+                $transPaths[] = $deprecatedPath = sprintf('%s/Resources/%s/translations', $kernel->getRootDir(), $bundle->getName());
+                if ($deprecatedPath !== $this->defaultTransPath.'/'.$bundle->getName() && is_dir($deprecatedPath)) {
```

Comment:
The overriding path for bundles shouldn't match `$this->defaultTransPath.'/'.$bundle->getName()` never, right?

Reply:
indeed, check removed